assets/js/hooks/handle_reaction_popup.js

```diff
@@ -0,0 +1,63 @@
+import { extractActionAndMessageIdFromDomElementId } from "../utils"
+
+function roundReactionPopupBorders(component) {
+    const style = document.createElement("style")
+    style.textContent = `
+        .picker {
+            border-radius: 20px;
+        }
+    `
+    component.el.shadowRoot.appendChild(style)
+}
```

Comment:
why do we need to create this dynamically?

Reply:
Because I can't do it with CSS directly, it's a requirement of the emoji-picker-element component, check this out: https://github.com/nolanlawson/emoji-picker-element